src/js/browser-plugin.js

```diff
@@ -138,24 +140,128 @@ const newFacebookPostCallback = (post) => {
   }
 };
 
-const classifyTweet = (tweet, score) => {
+const classifyTweet = (tweet, label) => {
 
-  const misinformationScore = score.misinformationScore;
   const node = tweet.domObject;
+  var myLabel = label.replace(/['"]+/g, '');
 
-  if (node.hasAttribute(parser.untrustedAttribute) && node.getAttribute(parser.untrustedAttribute)!=='undefined') {
-
+  if (node.hasAttribute(parser.untrustedAttribute) && node.getAttribute(parser.untrustedAttribute) !== 'undefined') {
     return;
+  } else {
+
+    var x = 0;
+    var a = configuration.coinform.misinformation;
+    for (i = 0; i < a.length; ++i) {
+      if (myLabel.localeCompare(a[i]) == 0) {
+        //if (myLabel.localeCompare('not verifiable post') != 0) 
+        node.setAttribute(parser.untrustedAttribute, 0);
+        node.append(whyCannotSeeTweetButton(tweet, label));
+        x = 1;
+      }
+    }
 
-  } else if (misinformationScore >= configuration.coinform.hideThreshold) {
-
-    node.setAttribute(parser.untrustedAttribute, misinformationScore);
-    node.append(createWhyButton('tweet'));
-    
+    if (x == 0) {
+      node.append(addCoinformLogo(tweet));
+    }
   }
-
 };
 
+const addCoinformLogo = (tweet) => {
+  var x = document.createElement("img");
+  x.setAttribute("src", 'https://en.gravatar.com/userimage/157153645/613b113e876d7942df3056fe07436977?size=200');
```

Comment:
Could we include this image in the plugin resources? Ideally we shouldn't have to make extra HTTP requests for this kind of stuff.

Reply:
I was getting a Cross-Origin Read Blocking (CORB) error while trying to access my local logo image thats why as a working-solution I implemented this.

Reply:
I just reimplemented it using the local resource icons.